test/metabase/api/search_test.clj

```diff
@@ -918,3 +930,114 @@
          (premium-features-test/with-premium-features #{}
            (is (= "Content Management or Official Collections is a paid feature not currently available to your instance. Please upgrade to use it. Learn more at metabase.com/upgrade/"
                   (mt/user-http-request :crowberto :get 402 "search" :q search-term :verified true)))))))))
+
+(deftest created-at-api-test
+  (let [search-term "created-at-filtering"]
+    (with-search-items-in-root-collection search-term
+      (testing "returns only applicable models"
+        (is (= #{"dashboard" "table" "dataset" "collection" "database" "action" "card"}
+               (-> (mt/user-http-request :crowberto :get 200 "search" :q search-term :created_at "today")
+                   :available_models
+                   set))))
+
+      (testing "works with others filter too"
+        (is (= #{"dashboard" "table" "dataset" "collection" "database" "action" "card"}
+               (-> (mt/user-http-request :crowberto :get 200 "search" :q search-term :created_at "today" :creator_id (mt/user->id :rasta))
+                   :available_models
+                   set))))
+
+      (testing "error if invalids created_at string"
+        (is (= "Failed to parse created at param: today~"
+               (mt/user-http-request :crowberto :get 400 "search" :q search-term :created_at "today~" :creator_id (mt/user->id :rasta))))))))
+
+(deftest created-at-correctness-test
+  (let [search-term "created-at-filtering"
+        now           #t "2023-05-04T10:00Z[UTC]"
```

Comment:
It is misleading to call this `now`, since it isn't actually now. Better to call it `new` (as opposed to old) I think.
```suggestion
        new           #t "2023-05-04T10:00Z[UTC]"
```